grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -29,9 +29,11 @@
 
 
 import static ai.grakn.test.GraknTestEnv.usingTinker;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.not;
 import static org.junit.Assume.assumeTrue;
 
-public class Tests{
+public class ReasoningTests {
```

Comment:
What is it testing in reasoning? How is it different from `ReasonerTest`? If you can't come up with a good name which I appreciate can be hard at least javadoc and describe the purpose of this test class.